index.js

```diff
@@ -127,6 +131,7 @@ module.exports = fp(function from (fastify, opts, next) {
       if (onResponse) {
         onResponse(this.request.raw, this, res.stream)
       } else {
+        this.context.errorHandler = errorHandler
```

Comment:
you can't do this :(, it will leak memory!

Reply:
Thanks, I'd meant to check on the best way to do this. I'm hoping this is the correct way:

https://github.com/fastify/fastify-reply-from/pull/141/commits/45bbd9da38389be8a84ff37f0c0df77902d08c6c